website/blog/2019-09-07-managing-multiples-projects-with-lerna-and-yarn-workspaces.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Managing multiples projects with Lerna and Yarn Workspaces
+author: Sergio Herrera
+authorURL: https://github.com/sergiohgz
+authorImageURL: https://avatars3.githubusercontent.com/u/14012309
+authorTwitter: sergiohgz
+---
+
+Verdaccio is a project with a big ecosystem, composed by several projects, each one with its own configurations and ways to manage. This fact transforms a simple maintenance operation, like updating a common dependency, into a real hard work.
+
+We saw a problem, these configurations entropry made harder to work with all the projects. So, we needed to simplify and unify them to make it easier. We need a **monorepo**.
```

Comment:
> We saw a problem, these configurations entropry made harder to work with all the projects. So, we needed to simplify and unify them to make it easier. We need a **monorepo**.

A long the dependencies were growing, we realized all configurations were hard to maintain for all projects. Furthermore, simplify and unify them would make much easier, our solution is a **monorepo**.

Reply:
Better what you say, but like before, instead of dependencies I prefer `number of projects`